src/Dfe.PlanTech.AzureFunctions/Mappings/RecommendationSectionMapper.cs

```diff
@@ -36,17 +33,8 @@ public override async Task PostUpdateEntityCallback(MappedEntity mappedEntity)
 
         if (existing != null)
         {
-            existing.Answers.AddRange(await _db.RecommendationSectionAnswers
-                                                .Where(recSecAnswer => recSecAnswer.RecommendationSectionId == incoming.Id)
-                                                .Select(recSecAnswer => recSecAnswer.Answer)
-                                                .Select(answer => answer!)
-                                                .ToListAsync());
-
-            existing.Chunks.AddRange(await _db.RecommendationSectionChunks
-                                                .Where(recSecChunk => recSecChunk.RecommendationSectionId == incoming.Id)
-                                                .Select(recSecChunk => recSecChunk.RecommendationChunk)
-                                                .Select(chunk => chunk!)
-                                                .ToListAsync());
+            await _db.RecommendationSectionAnswers.IgnoreQueryFilters().Where(recSecAnswer => recSecAnswer.RecommendationSectionId == existing.Id).Include(recSecAnswer => recSecAnswer.Answer).ToListAsync();
+            await _db.RecommendationSectionChunks.IgnoreQueryFilters().Where(recSecChunk => recSecChunk.RecommendationSectionId == existing.Id).Include(recSecChunk => recSecChunk.RecommendationChunk).ToListAsync();
```

Comment:
same question again, should this be `existing.Chunks = await _...`

Reply:
Closing due to discuss on thread